conf/DEV.conf

```diff
@@ -10,7 +10,7 @@ identity {
   apiToken = "subscriptions-dev-client-token"
 }
 
-subscriptions.url="https://sub.thegulocal.com"
+subscriptions.url="https://idapi.code.dev-theguardian.com"
 
```

Comment:
I don't think this is not the url you want to change. You probably want identity.baseUri